pyomo/contrib/mindtpy/config_options.py

```diff
@@ -161,11 +168,17 @@ def _get_MindtPy_config():
         description="add dual bound constraint to enforce the objective satisfies best-found dual bound",
         domain=bool
     ))
+    CONFIG.declare("heuristic_nonconvex", ConfigValue(
+        default=False,
+        description="use dual solution from the nlp solver and slack variables to add OA cuts for equality constraints.",
```

Comment:
Replace with "use dual solution from the NLP solver and slack variables to add OA cuts for equality constraints (Equality relaxation) and minimize the sum of the slack variables (Augmented Penalty)"